x-pack/plugins/synthetics/server/legacy_uptime/lib/requests/search/get_query_string_filter.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const getQueryStringFilter = (query: string) => {
+  let queryString = query;
+  if (isAlphaNumeric(query) && !includesOperator(query.toLowerCase())) {
+    // if user doesn't specify any query string syntax we user wildcard buy default
+    queryString = `*${query}*`;
+  }
+
+  return {
+    query_string: {
+      query: queryString,
+      fields: ['monitor.id.text', 'monitor.name.text', 'url.full.text'],
+    },
+  };
+};
+
+const includesOperator = (query: string) => {
+  return query.includes(' or ') || query.includes(' and ');
+};
+
+export const isAlphaNumeric = (str: string) => {
```

Comment:
We seem to be duplicating the function declared [above](https://github.com/elastic/kibana/pull/145533/files#diff-d735129605cdd76779862493b4dea192a16c6dc70a6d05086a9fe754baf1bf0fR51). Why not just reference that function?

Reply:
It's been noted to me that this one contains a different regex value. Perhaps we can adopt some better names for these functions to make their intentions clearer.